apps/src/aichat/redux/utils.ts

```diff
@@ -78,3 +80,28 @@ export const allFieldsHidden = (fieldVisibilities: FieldVisibilities) =>
   getTypedKeys(fieldVisibilities).every(
     key => fieldVisibilities[key] === Visibility.HIDDEN
   );
+
+// Opposite of decorateMessageFromModelResponse.
+// Preps messages to send to model for response.
+export const trimMessageForModelInput = (
```

Comment:
I can see what you're saying about the tricky naming of this function. 

Nit: 'trimMessage` seems to imply that the message text/content is trimmed. What about `prepMessageForModelInput`? I don't feel strongly either way - just another option.